rca/utils/blocks.py

```diff
@@ -7,6 +7,17 @@
 from wagtail.snippets.blocks import SnippetChooserBlock
 
 
+class FeeBlock(blocks.StructBlock):
+    location = blocks.CharBlock(max_length=120, help_text="e.g. Home and EU")
+    subsidised = blocks.BooleanBlock(required=False)
+    per_year_cost = blocks.CharBlock(
+        max_length=120, help_text="e.g. £14,200 per year", required=False
+    )
+    total_cost = blocks.CharBlock(
```

Comment:
I feel weird about not using number fields for this. You can achieve the number formatting with https://docs.djangoproject.com/en/2.2/ref/contrib/humanize/#intcomma . It will result in editors filling cost in different formats.

Reply:
I share your concern, the reason I have built this with text fields as so the values can be per month, per year, per term. Again I think it will work better using number fields and snippets for the  'per year/month' values but this won't be in this phase